src/Bundle/Resources/config/services/routing.xml

```diff
@@ -99,10 +103,14 @@
         <service id="sylius.routing.factory.route_attributes" class="Sylius\Bundle\ResourceBundle\Routing\RouteAttributesFactory" public="false" />
         <service id="Sylius\Bundle\ResourceBundle\Routing\RouteAttributesFactoryInterface" alias="sylius.routing.factory.route_attributes" />
 
-        <service id="sylius.routing.factory.attributes_operation_route" class="Sylius\Resource\Symfony\Routing\Factory\AttributesOperationRouteFactory" public="false">
+        <service id="sylius.routing.factory.attributes_operation_route"
+                 class="Sylius\Resource\Symfony\Routing\Factory\AttributesOperationRouteFactory"
+                 public="false"
+        >
             <argument type="service" id="sylius.resource_registry" />
             <argument type="service" id="sylius.routing.factory.operation_route" />
             <argument type="service" id="sylius.resource_metadata_collection.factory.attributes" />
+            <deprecated package="sylius/resource-bundle" version="1.13">The "%service_id%" service is deprecated since sylius/resource-bundle 1.13 and will be removed in sylius/resource-bundle 2.0. Use sylius.routing.resource.route_collection_factory instead.</deprecated>
```

Comment:
I'm not sure about the "%service_id%" param, I need to check that.

Reply:
https://symfony.com/doc/current/service_container/alias_private.html#deprecating-services